docker-compose.yml

```diff
@@ -22,17 +40,15 @@
     build:
       context: .
       dockerfile: Dockerfile.mysql
-    container_name: my-sql
+    container_name: mysql
     ports:
       - "3306:3306"
     volumes:
       - todoapp-db-data:/var/lib/mysql
     environment:
-      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_ROOT_PASSWORD=99887766aA
```

Comment:
The `MYSQL_ROOT_PASSWORD` is hardcoded with a specific value. This poses a security risk. Consider using Docker secrets or setting this variable at runtime to protect sensitive information.